chain/cosmos/chain_node.go

```diff
@@ -479,17 +569,9 @@ type CosmosTx struct {
 }
 
 func (tn *ChainNode) SendIBCTransfer(ctx context.Context, channelID string, keyName string, amount ibc.WalletAmount, timeout *ibc.IBCTimeout) (string, error) {
-	command := []string{tn.Chain.Config().Bin, "tx", "ibc-transfer", "transfer", "transfer", channelID,
+	command := []string{
+		"ibc-transfer", "transfer", "transfer", channelID,
```

Comment:
Out of scope for this PR. But should we concerned about hard-coding the src-port of "transfer" ?

Reply:
Only transfer port for now. A future need for different ports might involve a refactor to this method or a new method.